manifests/stable/dpu-network-operator.clusterserviceversion.yaml

```
@@ -33,6 +33,8 @@ metadata:
     operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
     repository: https://github.com/openshift/dpu-network-operator
     support: Red Hat
+  labels:
+    operatorframework.io/arch.arm64: supported
   name: dpu-network-operator.v0.0.1
```

this needs to match what art.yaml says to replace, which is currently `dpu-network-operator.v4.10.0`. i'd suggest updating art.yaml to match what's here instead, then a change won't be necessary for the next branch. up to you though, this works too:
```suggestion
  name: dpu-network-operator.v4.10.0
```